chain/chain/src/store.rs

```diff
@@ -1128,6 +1141,19 @@ impl<'a> ChainStoreAccess for ChainStoreUpdate<'a> {
         }
     }
 
+    /// The chain tail height, used by GC.
+    fn tail(&self) -> Result<BlockHeight, Error> {
```

Comment:
Since this function never returns error, why not just return `BlockHeight`?

Reply:
I want to keep compatibility with ChainStoreAccess, and chain_store.tail() executes `option_to_not_found` which returns Result.